include/rmm/mr/host/pinned_memory_resource.hpp

```diff
@@ -99,5 +172,6 @@ class pinned_memory_resource final : public host_memory_resource {
       ptr, bytes, alignment, [](void* ptr) { RMM_ASSERT_CUDA_SUCCESS(cudaFreeHost(ptr)); });
   }
 };
+static_assert(cuda::mr::async_resource<pinned_memory_resource>);
```

Comment:
question: Should we test `async_resource_with<pinned_memory_resource, host_accessible, device_accessible>`?

Reply:
So currently it is not supporting device_accessible, we should probably change that